### DISCUSSION TITLE
Can you submit a pull request to selfhosted_templates?

### DISCUSSION THREAD

**Discussion by JoshuaMunoz** [socioboard/Socioboard-5.0#271] [Ideas]
https://github.com/SelfhostedPro/selfhosted_templates

These guys have created an amazing, one-click deployment for a number of tools that run in Docker. I would love for Socioboard (open source) to be added, but they're asking for you to submit a pull request to it.

Please let me know. Thank you.

**Comment by sureshbabu1995** [comment]
> https://github.com/SelfhostedPro/selfhosted_templates
> 
> These guys have created an amazing, one-click deployment for a number of tools that run in Docker. I would love for Socioboard (open source) to be added, but they're asking for you to submit a pull request to it.
> 
> Please let me know. Thank you.

Hi, That's a valid request, but we need to make our services for Docker, soon we will update the code for docker support, then we'll make it much easier as you suggest.
